### DISCUSSION TITLE
Solution structure

### DISCUSSION THREAD

**Discussion by jtreher** [8T4/c4sharp#24] [General]
@yanjustino Hello, 

On my PR, I pointed out that I had started working on the unit tests and adding functionality: https://github.com/jtreher/c4sharp/tree/add-tests 

However, the commit just added code completely restructured the project. Is there any reason we couldn't use the standard layout recommended by David Fowler? https://gist.github.com/davidfowl/ed7564297c61fe9ab814

If you could revert your change and I merge mine it, we should be good to go. I can copy in your stubs to my PR as well.

We'd keep src/ as it was and tests/ would be a sibling folder.

Best,

Justin

**Comment by yanjustino** [comment]
Thank you, @jtreher. You have an excellent point! Next release I'll fix the solution structure as you say!
